database/seed.js

```diff
@@ -0,0 +1,32 @@
+const db  = require('./index.js');
+
+
+const samplePosts = [
+  {
+    uniqId: 123,
+    symbol: 'aapl',
+    name: 'Apple',
+    marketPrice: 100,
+    created_at: "2017-05-03T03:50:00Z",
```

Comment:
You may use single quotation for consistency
```suggestion
    created_at: '2017-05-03T03:50:00Z',
```